Modules/Core/Common/wrapping/itkFunctionBase.wrap

```diff
@@ -28,7 +28,15 @@ itk_wrap_class("itk::FunctionBase" POINTER)
 
     # Required by BSplineInterpolationWeightFunction
     itk_wrap_template("${ITKM_CID2}${ITKM_FAD9}" "${ITKT_CID2}, ${ITKT_FAD9}")
```

Comment:
Shouldn't this line be replaced by the below loop too?

Reply:
Looks like the loop is for cubic splines.  Looks like this one line adds quadratic splines for 2-dimensional images.

Reply:
This was needed, but it turns out that there was a mistake in the wrapping of BSplineInterpolationWeightFunction SplineOrder.